doc/api/fs.md

```diff
@@ -1298,6 +1303,7 @@ default value of 64 kb for the same parameter.
 
 ```js
 const defaults = {
+  end: Infinity,
```

Comment:
@ryzokuken `start` is set to `0` by default if I am not mistaken (implicit).

@vsemozhetbyt seems like the documentation of `fs` is not aligned with most other documentations when it comes to e.g. default values and return types. Is there an issue to track and improve this?

Reply:
@BridgeAR The default for `start` is `undefined`, i.e. reading from the current position in the file

Reply:
@BridgeAR exactly as @addaleax said. If `start` is unset, we're not setting it to any default value.

Reply:
@BridgeAR I will try.

Reply:
@vsemozhetbyt I could help if you need any.

Reply:
I will post a PR link here, you can review)

But there may be conflicts with this PR as I plan to incorporate objects with defaults into parameter lists an delete the objects. You or I will need to rebase)

Reply:
@BridgeAR, @ryzokuken, PTAL: https://github.com/nodejs/node/pull/20050